airflow/contrib/operators/app_engine_operator.py

```diff
@@ -1,9 +1,12 @@
+from airflow import configuration
 from airflow.contrib.hooks.gcs_hook import GoogleCloudStorageHook
-from airflow.exceptions import AirflowException
+from airflow.exceptions import AirflowException, AirflowTaskTimeout
```

Comment:
Turns out Airflow already had an exception for timeouts. This saves me the trouble of making one and still avoids conflict with Python 3's [`TimeoutError`](https://docs.python.org/3/library/exceptions.html#TimeoutError)